app/api/stripe_authorization.py

```diff
@@ -69,7 +70,8 @@ def after_create_object(self, stripe_authorization, data, view_kwargs):
         save_to_db(event)
 
     schema = StripeAuthorizationSchema
-    decorators = (jwt_required, )
+    decorators = (api.has_permission('is_coorganizer', fetch="event_id",
+                                     fetch_as="event_id", model=StripeAuthorization),)
```

Comment:
https://github.com/fossasia/open-event-frontend/issues/3523 is happening due to this. The endpoint is `/v1/stripe-authorizations`, there is no `event_id` in view kwargs and hence it fails everytime it is accessed

Reply:
@CosmicCoder96 Can this be removed or was it added for a specific reason. I can't see how it will work for /v1/stripe-authorizations endpoint. Maybe it is used in some other relations. Please clarify so that we don't break something else when removing this

Reply:
@iamareebjamal It should be removed. Won't affect FE.